### DISCUSSION TITLE
Unpublished whole package but did specify just specific version (Bug)?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [community/community#124401] [npm]
In my case it was not even the last version, I had published the package accidentally with npm's default version 1.0.0 then I added another version 0.0.1 to fix that because I like to start packages that way, and then I unpublished version 1.0.0 so that only 0.0.1 stays but the whole package was removed, this seems like a bug to me?

**Comment by dinith-rusiru** [comment]
Response 1#
